src/tests-rosetta/api.ts

```diff
@@ -1074,6 +1090,717 @@ describe('Rosetta API', () => {
+      network_identifier: {
+        blockchain: 'stacks',
+        network: 'testnet',
+      },
+      operations: [
+        {
+          operation_identifier: {
+            index: 0,
+            network_index: 0,
+          },
+          related_operations: [],
+          type: 'fee',
+          status: 'success',
+          account: {
+            address: sender,
+            metadata: {},
+          },
+          amount: {
+            value: fee,
+            currency: {
+              symbol: 'STX',
+              decimals: 6,
+            },
+            metadata: {},
+          },
+        },
+        {
+          operation_identifier: {
+            index: 1,
+            network_index: 0,
+          },
+          related_operations: [],
+          type: 'token_transfer',
+          status: 'success',
+          account: {
+            address: sender,
+            metadata: {},
+          },
+          amount: {
+            value: '-500000',
+            currency: {
+              symbol: 'STX',
+              decimals: 6,
+            },
+            metadata: {},
+          },
+        },
+        {
+          operation_identifier: {
+            index: 2,
+            network_index: 0,
+          },
+          related_operations: [],
+          type: 'token_transfer',
+          status: 'success',
+          account: {
+            address: recipient,
+            metadata: {},
+          },
+          amount: {
+            value: '500000',
+            currency: {
+              symbol: 'STX',
+              decimals: 6,
+            },
+            metadata: {},
+          },
+        },
+      ],
+      public_keys: [
+        {
+          hex_bytes: publicKey,
+          curve_type: 'secp256k1',
+        },
+      ],
+    };
+
+    const accountInfo = await new StacksCoreRpcClient().getAccount(sender);
+
+    const tokenTransferOptions: UnsignedTokenTransferOptions = {
+      recipient: recipient,
+      amount: new BN('500000'),
+      fee: new BN(fee),
+      publicKey: publicKey,
+      network: GetStacksTestnetNetwork(),
+      nonce: accountInfo.nonce ? new BN(accountInfo.nonce) : new BN(0),
+    };
+
+    const transaction = await makeUnsignedSTXTokenTransfer(tokenTransferOptions);
+    const unsignedTransaction = transaction.serialize();
+    const hexBytes = digestSha512_256(unsignedTransaction).toString('hex');
+
+    const result = await supertest(api.server)
+      .post(`/rosetta/v1/construction/payloads`)
+      .send(request);
+
+    expect(result.status).toBe(200);
+    expect(result.type).toBe('application/json');
+
+    const expectedResponse = {
+      unsigned_transaction: unsignedTransaction.toString('hex'),
+      payloads: [
+        {
+          address: sender,
+          hex_bytes: '0x' + hexBytes,
+          signature_type: 'ecdsa',
+        },
+      ],
+    };
+
+    expect(JSON.parse(result.text)).toEqual(expectedResponse);
+  });
+
+  test('payloads multi sign success', async () => {
```

Comment:
typo? Test looks like `payloads multi sig success`, the test is only unsigned txs

Reply:
No,  It creates unsigned txs with multiple signatures.